core/src/main/java/io/kestra/core/models/tasks/NamespaceFilesInterface.java

```diff
@@ -0,0 +1,13 @@
+package io.kestra.core.models.tasks;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public interface NamespaceFilesInterface {
+    @Schema(
+        title = "Inject namespace files",
+        description = "Inject namespace file on this task. Inject all namespaces files when enabled or use filter to limit injected files."
```

Comment:
```suggestion
        description = "Inject namespace files to this task. When enabled, it will load all namespace files into the working directory by default. However, you can use the `include` or `exclude` properties to limit which namespace files will be injected."
```